README.md

```diff
@@ -15,17 +15,20 @@ pip install --upgrade postgrid-python
 
 ### Requirements
 
-- Python 3.6+
+-   Python 3.6+
```

Comment:
Any particular reason for the additional spaces here or was it an autoformatting thing?

Reply:
Yeah prettier kept adding spaces, removed it anyways.